cmd/skywire-visor/commands/root.go

```diff
@@ -97,6 +92,8 @@ func init() {
 		rootCmd.Flags().BoolVarP(&launchBrowser, "browser", "b", false, "open hypervisor ui in default web browser")
 	}
 	rootCmd.Flags().BoolVarP(&hypervisorUI, "hvui", "i", false, "run as hypervisor")
+	rootCmd.Flags().BoolVarP(&noHypervisorUI, "nohvui", "x", false, "disable hypervisor")
```

Comment:
Whats this doing now? Ad-hoc disabling the hypervisorUI? I guess this is useful for you somehow...

Reply:
Yes, i needed that for testing purposes.